BEIMA.Backend.FT/BuildingFT.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MongoDB.Bson;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using static BEIMA.Backend.FT.TestObjects;
+
+namespace BEIMA.Backend.FT
+{
+    [TestFixture]
+    public class BuildingFT : FunctionalTestBase
+    {
+        [Test]
+        public async Task BuildingNotInDatabase_AddBuilding_CreatesNewBuilding()
+        {
+            // ARRANGE
+            var building = new Building
+            {
+                Name = "Student Union",
+                Number = "1234",
+                Notes = "Some notes.",
+                Location = new Location
+                {
+                    Longitude = "123.001",
+                    Latitude = "101.321",
```

Comment:
This lat value out of range, do we need checking for that on the backend, or is the front end validation enough?

Reply:
Haven't added the checking for the backend on this yet, I think we are going to do that on another task, but I will update this to values that are actually in range.